### ISSUE TITLE
In Submodule update - "init" to be disabled by default

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#2293]
```
What steps will reproduce the problem?
1. TortoiseGIT | Submodule Update
2. Initialization is by default turned on

What is the expected output? What do you see instead?
Initialization to be turned off by default. It is usually used only once and 
then many times is done update without initialization. I have to turned each 
time I do submodule update now.


What version of TortoiseGit and msysgit are you using? On
what operating system?
Windows 8.1 x64, TortoiseGIT 1.8.11

Please provide any additional information below.

```

Original issue reported on code.google.com by `var...@gmail.com` on 11 Sep 2014 at 1:31
- Merged into: #2294
